Boccia-Unity/Assets/Boccia/UI/StartMenuPresenter.cs

```diff
@@ -18,11 +18,11 @@ void Start()
         // cache model and subscribe for changed event
         model = BocciaModel.Instance;
         BocciaModel.WasChanged += ModelChanged;
```

Comment:
I know you didn't change this in this PR, but shouldn't this be model.WasChanged instead of BocciaModel.WasChanged?

Reply:
Yeah, this has to be BocciaModel because I made the event static.  But, as we discussed, I'm not certain it needs to be static.  I'll look into why I did it that way.